carbon-relay-ng.go

```diff
@@ -315,8 +479,17 @@ func main() {
 		go HttpListener(config.Http_addr, table)
 	}
 
-	if err := goagain.AwaitSignals(l); nil != err {
-		log.Error(err.Error())
-		os.Exit(1)
+	if nil != l {
```

Comment:
this sort of trickery is not needed in Go, because the mistake that this tries to protect from won't be accepted by the compiler. (see https://play.golang.org/p/Gu726BUiJx for example)

anyway, not a merge-showstopper